### ISSUE TITLE
Error: unrecognized cop or department RSpecRails/HttpStatus found in config/default.yml Did you mean `RSpecRails/HaveHttpStatus`? 

### ISSUE THREAD

**Issue by rubyonrails3** [rubocop/rubocop-rspec#1846]
I am getting this error, today suddenly my CI stopped working, as my Gemfile.lock is not part of the gem and it installs the latest available gems. 

Error: unrecognized cop or department RSpecRails/HttpStatus found in /Users/ismail/.rvm/gems/ruby-3.3.0/gems/rubocop-rspec_rails-2.28.1/config/default.yml
Did you mean `RSpecRails/HaveHttpStatus`?

Any idea about it?

**Comment by mjankowski** [comment]
Seeing similar issue (using `rubocop-rspec` 2.28.0 and `rubocop-rspec_rails` 2.28.1). The release notes claim:

> the cops related to rspec-rails are aliased (RSpec/Rails/Foo == RSpecRails/Foo) until v3.0 is released, so the change will be invisible to users until then

I wonder if the aliasing was not included in the release ... or a decision was made to not do the aliasing and the release notes are wrong?

**Comment by ntkme** [comment]
It looks like this is an issue because `RSpecRails/HaveHttpStatus` is conditionally loaded, and do not exist if `rack` is not installed: https://github.com/rubocop/rubocop-rspec_rails/blob/b141576cf6fb1f75bb3e446b96d4c4f8266cc161/lib/rubocop/cop/rspec_rails_cops.rb#L6-L10

```
begin
  require_relative 'rspec_rails/http_status'
rescue LoadError
  # RSpecRails/HttpStatus cannot be loaded if rack/utils is unavailable.
end
```

**Comment by ydah** [comment]
@rubyonrails3 @mjankowski @ntkme Thank you for your reporting. I have created a PR for the fix below. Does the following solve it?
- https://github.com/rubocop/rubocop-rspec/pull/1849

**Comment by ntkme** [comment]
@ydah I think rack might be add as a dependency for `rubocop-rspec_rails`, but not `rubocop-rspec`. I don't use rails in my repo thus I don't think rack is needed for me.

**Comment by rubyonrails3** [comment]
@ydah  yes it does fix the issue for me. just tried this `gem "rubocop-rspec", github: "rubocop/rubocop-rspec", ref: "198db78eef099ad605705ebce038ad3ba71ad6ef"` and everything worked fine
